made_with_lush/README.md

```diff
@@ -24,6 +24,22 @@ A port of the jellybeans theme.
 
 ![first screenshot](metalelf0_jellybeans-nvim_1.png)
 
+---
+### onedark.nvim
+
+[olimorris/onedark.nvim](https://github.com/olimorris/onedark.nvim)
+
```

Comment:
Thanks for the PR! Can you fix up this part of the readme to match the template: including the lua module name?

`require(<what you require in your colours/vim file, probably 'lush_theme.one_dark'>)`

Reply:
Ahhh sorry i've perhaps misinterpreted your initial ask. Have updated now.

Reply:
Nah, really you just made me realize that the template is missing some bits.